### ISSUE TITLE
Consider alternatives to the URL versioning scheme

### ISSUE THREAD

**Issue by Lomilar** [cassproject/CASS#34]
RDF does not guarantee HTTP based retrieval mechanisms. Therefore, the effective method of creating redirects for HTTP requests to link from a typeless shortId or shortId (/data/<guid> or /data/<type>/<guid>) to the @id (/data/<type>/<guid>/<tsinMils>) as links in data can cause issues.

This makes some CASS data incompatible with other RDF tools.

**Comment by Lomilar** [comment]
Impact: This cannot be retroactively applied to 0.3 or below objects, due to invalidation of signatures. Upgrade path is probably the best way to accomplish it.

**Comment by Lomilar** [comment]
Note: Hardly anything uses the exact version signature. Exact version signatures continue to need to exist in order to linked-data reference to a specific version of data.

**Comment by Lomilar** [comment]
eduworks/ew#9

**Comment by Lomilar** [comment]
https://medium.com/@lomilar/linked-open-data-dont-make-the-http-assumption-if-you-can-help-it-b0dc791a3b8a

@stuartasutton I'm at a bit of an impasse here. I think for LD on the Web, no system should presume that a URL reference whether "<url>" or {@id:"<url>"} will resolve to a resource with the same @id. So, as part of an export process from LD on the web to RDF on a machine, one needs to normalize all the links to the @id of what an HTTP GET of the @id resolves to.

In short, on the web, pointers (304 redirects) exist, and I can't find a way around them existing...

Thoughts?

**Comment by Lomilar** [comment]
Sidestepped by allowing CRUD+Search of non-versioned (@id does not point to CASS) objects. Disabled versioning for CRUD+Search of registered objects.
